control-plane/agents/src/bin/core/tests/volume/capacity_limit.rs

```diff
@@ -0,0 +1,88 @@
+#![cfg(test)]
+
+use deployer_cluster::ClusterBuilder;
+use grpc::operations::volume::traits::VolumeOperations;
+use std::time::Duration;
+use stor_port::{
+    transport_api::ReplyErrorKind,
+    types::v0::transport::{CreateVolume, DestroyVolume},
+};
+use uuid::Uuid;
+
+const SIZE: u64 = 5242880;
+const EXCESS: u64 = 1000000;
+
+#[tokio::test]
+async fn volume_create_with_capacity_limit() {
+    let cache_period = Duration::from_millis(250);
+    let reconcile_period = Duration::from_millis(3000);
+    let cluster = ClusterBuilder::builder()
+        .with_rest(true)
```

Comment:
We don't use rest here?

```suggestion
        .with_rest(false)
```

Reply:
fixed